python/ray/train/tensorflow/tensorflow_predictor.py

```diff
@@ -21,23 +21,38 @@ class TensorflowPredictor(Predictor):
         preprocessor: A preprocessor used to transform data batches prior
             to prediction.
         model_weights: List of weights to use for the model.
+        use_gpu: If prediction is done on GPU where model will be moved to
+            GPU device upon instantiation.
     """
 
     def __init__(
         self,
         model_definition: Union[Callable[[], tf.keras.Model], Type[tf.keras.Model]],
         preprocessor: Optional["Preprocessor"] = None,
         model_weights: Optional[list] = None,
+        use_gpu: bool = False,
     ):
         self.model_definition = model_definition
         self.model_weights = model_weights
         self.preprocessor = preprocessor
 
+        self.use_gpu = use_gpu
```

Comment:
Can we raise a loud warning if use_gpu is set to False but a GPU device is detected (see the RLlib code for detecting GPUs).